src/plugins/guided_onboarding/public/components/guide_panel.styles.ts

```diff
@@ -43,5 +43,15 @@ export const getGuidePanelStyles = (euiTheme: EuiThemeComputed) => ({
       background: ${euiTheme.colors.ghost};
       padding: 24px 30px;
     `,
+    flyoutFooterLink: css`
+      color: ${euiTheme.colors.darkShade};
+      font-weight: 400;
+
+      svg[data-icon-type='questionInCircle'] {
```

Comment:
I wouldn't expect we would have to override the default styles for [buttons with icons](https://elastic.github.io/eui/#/navigation/button#buttons-with-icons). Can you provide more context? Thanks!

Reply:
For sure, sorry if I was unclear. By default, the icons in the buttons appear mismatched.
![image](https://user-images.githubusercontent.com/11224465/197550525-0cf76390-faa0-495c-ad4f-29634ca98970.png)

I tried to play around with the size prop for each button to see if I could get them to match visually but found the best way was to add a custom style to the "Need help?" icon as it is the one that appears most mismatched. 

Let me know if you disagree or think it should be done a different way!

Reply:
I agree with @alisonelizabeth that we should be careful with overriding any default styles. In this case, I think we should open an issue with EUI for the icons size so that it could be fixed ideally at the source. The link to the issue we could add to a code comment describing why the custom css is needed here.